content/en/logs/troubleshooting/_index.md

```diff
@@ -4,13 +4,19 @@ title: Logs Troubleshooting
 
 If you experience unexpected behavior with Datadog Logs, there are a few common issues you can investigate and this guide may help resolve issues quickly. If you continue to have trouble, reach out to [Datadog support][1] for further assistance.
 
+## Missing logs - data access restrictions
+
+You cannot see any logs neither in the [Log Explorer][2] or [Live Tail][3]. This may be happening because your role is part of a restriction query.
```

Comment:
```suggestion
You cannot see any logs in the [Log Explorer][2] or [Live Tail][3]. This may be happening because your role is part of a restriction query.
```